src/nft/ZoraCreator1155Impl.sol

```diff
@@ -623,6 +698,25 @@ contract ZoraCreator1155Impl is
         }
     }
 
+    /// @notice Withdraws ETH from the Zora Rewards contract
+    function withdrawRewards(uint256 amount) public onlyAdminOrRole(CONTRACT_BASE_ID, PERMISSION_BIT_FUNDS_MANAGER) {
```

Comment:
does this need to be only called by an admin or `PERMISSION_BIT_FUNDS_MANAGER` since it always withdraws to the fundsRecipient?